src/table/table.scss

```diff
@@ -4,383 +4,423 @@
-            @include themed {
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+  // Sort icon
+  .iui-sort {
+    visibility: hidden;
+    margin-left: auto;
+    width: $iui-icons-default;
+    height: $iui-icons-default;
+    @include themed {
+      fill: t(iui-icons-color-actionable);
+    }
+  }
+
+  // Sorted column
+  .iui-sorted {
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
 
-        > .iui-filter {
-          margin-left: $iui-sm;
-          cursor: pointer;
+    .iui-sort {
+      visibility: visible;
+    }
+  }
 
-          > .iui-icon {
-            width: $iui-icons-default;
-            height: $iui-icons-default;
-          }
-        }
+  // Double header
+  .iui-row + .iui-row {
+    @include themed {
+      border-top: 1px solid t(iui-color-background-5);
+    }
+  }
+}
 
-        > .iui-sort {
-          flex-grow: 1;
-          justify-content: flex-end !important;
+@mixin iui-table-body {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  
+  .iui-row {
+    display: flex;
+    flex-grow: 1;
+    margin-top: -1px;
+    border: solid 1px transparent;
+    @include themed {
+      border-bottom-color: t(iui-color-background-4);
+    }
 
-          > .iui-icon-wrapper {
-            cursor: pointer;
+    > .iui-slot > .iui-more-options {
+      visibility: hidden;
+    }
 
-            > .iui-icon {
-              width: $iui-icons-default;
-              height: $iui-icons-default;
-            }
-          }
+    &:hover:not(.iui-disabled) {
+      &:not(.iui-selected):not(.iui-expandable-content) {
+        @include themed {
+          background-color: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-6)
+          );
         }
+      }
 
-        > .iui-filter,
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          height: $iui-icons-default;
-          width: $iui-icons-default;
-          display: flex;
-          justify-content: center;
-          align-items: center;
-          visibility: hidden;
-        }
+      > .iui-slot > .iui-more-options {
+        visibility: visible;
+      }
+    }
 
-        > .iui-filter.iui-active {
-          visibility: visible;
-  
-          > .iui-icon {
-            @include themed {
-              fill: t(iui-icons-color-primary);
-            }
-          }
-        }
+    &:focus-within > .iui-slot > .iui-more-options {
+      visibility: visible;
+    }
 
-        &:hover {
-          @include themed {
-            background-color: t(iui-color-background-4);
-          }
+    // #region Selection
+    &:not(.iui-selected) + .iui-selected {
+      padding-bottom: $iui-xxs;
+      border-bottom: none;
+    }
 
-          > .iui-filter,
-          > .iui-sort,
-          > .iui-sort > .iui-icon-wrapper {
-            visibility: visible;
-          }
-        }
+    &.iui-selected {
+      padding: 0;
```

Comment:
I could not find the case, where this padding would be needed to override something...

Reply:
This line is good, the line 208 is wrong. (& + & { padding-top: 0; })